packages/smooth_app/pubspec.yaml

```diff
@@ -18,12 +18,14 @@ dependencies:
   flutter_staggered_animations: ^1.0.0
   flutter_staggered_grid_view: ^0.4.1
   flutter_svg: ^0.23.0+1
+  flutter_map: any
```

Comment:
I'd prefer a real version number rather than `any`.

Reply:
For sure, this was just experimenting, and forgot to update, thanks for catching